src/litDataset.ts

```diff
@@ -84,15 +85,20 @@ export async function fetchLitDataset(
   const resourceInfo = parseResourceInfo(response);
 
   const resourceWithResourceInfo: LitDataset &
-    ResourceInfo = Object.assign(resource, { resourceInfo: resourceInfo });
+    ResourceWithInfo = Object.assign(resource, { resourceInfo: resourceInfo });
 
   return resourceWithResourceInfo;
 }
 
 /**
- * @internal
+ * Retrieve the information about a resource (e.g. access permissions) without
+ * fetching the resource itself.
+ *
+ * @param url URL to fetch a [[LitDataset]] from.
+ * @param options Optional parameter `options.fetch`: An alternative `fetch` function to make the HTTP request, compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters).
+ * @returns Promise resolving to a [[LitDataset]] containing the data at the given Resource, or rejecting if fetching it failed.
```

Comment:
This doesn't seem right - it doesn't return a LitDataset, right?

```suggestion
 * @param url URL to fetch Resource metadata from.
 * @param options Optional parameter `options.fetch`: An alternative `fetch` function to make the HTTP request, compatible with the browser-native [fetch API](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch#Parameters).
 * @returns Promise resolving to the metadata describing the given Resource, or rejecting if fetching it failed.
```

(Btw, one other error that I sneaked in earlier: the "Parameters" heading on the MDN docs page has an ID of `#Parameters`, not `#parameters`. Don't know how that got in my browser history. Also, it'll likely be called otherwise in different languages, so I did include the locale in the URL here - though I'm not sure whether it's better to end up at the top of the page in your own language, or at the right section in English...)

Reply:
I'd say the right section is more appropriate, and our whole documentation is in english anyways, so including the locale doesn't seem problematic.